cpp/tests/groupby/argmin_tests.cpp

```diff
@@ -91,6 +91,26 @@ TYPED_TEST(groupby_argmin_test, zero_valid_values)
   test_single_agg(keys, vals, expect_keys, expect_vals, std::move(agg2), force_use_sort_impl::YES);
 }
 
+TYPED_TEST(groupby_argmin_test, zero_valid_values_xx)
```

Comment:
Please create a separate test such as `groupby_minby_test`, do not use existing test for easier debugging.